index.html

```diff
@@ -1075,6 +1078,115 @@ <h2>Status List Verifiable Credential</h2>
+        </p>
+
+        <pre class="example nohighlight vc"
+             title="Associating multiple status lists with a single Verifiable Credential"
+             data-vc-vm='https://example.edu/issuers/565049/keys/1'>
+{
+  "@context": [
+    "https://www.w3.org/ns/credentials/v2",
+    "https://www.w3.org/ns/credentials/examples/v2"
+  ],
+  "id": "https://example.com/credentials/23894672394",
+  "type": ["VerifiableCredential"],
+  "issuer": "did:example:12345",
+  "issuanceDate": "2021-04-05T14:27:42Z",
+  <span class="comment">// note the use of an array to represent the set of
+  // status entries</span>
+  "credentialStatus": <span class="highlight">[{
+    "id": "https://example.com/credentials/status/3#94567",
+    "type": "BitstringStatusListEntry",
+    "statusPurpose": "revocation",
+    "statusListIndex": "94567",
+    "statusListCredential": "https://example.com/credentials/status/3"
+  }, {
+    "id": "https://example.com/credentials/status/4#12345",
+    "type": "BitstringStatusListEntry",
+    "statusPurpose": "suspension",
+    "statusListIndex": "12345",
+    "statusListCredential": "https://example.com/credentials/status/4"
+  }]</span>,
+  "credentialSubject": {
+    "id": "did:example:6789",
+    "type": "Person"
+  }
+}
+        </pre>
+      </section>
+
+      <section>
+        <h2>Multiple Status Entries in a Single List</h2>
+
+        <p>
+It is possible for a single status list to contain multiple types of status
+purposes. Doing so can make the retrieval of a list slightly more efficient
+than fetching multiple status lists.
+        </p>
+
+        <p class="issue atrisk" title="Efficiency argument is weak">
+The "space efficiency" argument for this feature is weak. Fetching one list
+with two types of status entries must, presumably, be twice as long to ensure
+proper privacy protections. One privacy benefit of doing so is that bit flips
+cannot be known to be associated with a particular status unless one is also in
+control of the VC that the status is about. Therefore, mixing "revocation" and
+"suspension" in a single list that is twice as large has positive privacy
+implications.<br><br>
+The "retrieval efficiency" argument is also weak. Performing two HTTP retrievals
+instead of one is probably not significant. Performing upwards of five to six,
+on a list that is five to six times larger, might result in fairly meager
+savings over modern versions of HTTP that bundle requests over a single channel
+(such as HTTP/2 or HTTP/3). The requests themselves would be a handful of bytes
+that are saved with no significant improvement in retrieval speed.<br><br>
+The Working Group is looking for feedback from implementers and is considering
+striking this feature during the Candidate Recommendation period since it would
+simplify the specification for implementations to not have to support sets of
+`proofPurpose` values in the status list credentials (again, a meager savings in
+complexity).
```

Comment:
`a meager savings in complexity` seems to go against the rest of this note, which discusses meager savings in time and/or data size, gained at the expense of more complex implementations...
```suggestion
`proofPurpose` values in the status list credentials (again, a meager savings in
complexity).
```